### PR TITLE
Divyanshu | Stage |  Support of excel export in other voucher get all page

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cuzbbu5

### CODE REVIEW

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 1]
```diff
@@ -93,32 +93,43 @@
...
                 >
                     <button
-                        dropdownToggle
-                        type="button"
-                        class="btn btn-primary more-btn-dropdown dropdown-toggle mr-0"
-                        aria-controls="dropdown-basic"
+                        class="btn btn-default mr-r15"
+                        [tooltip]="commonLocaleData?.app_export"
+                        (click)="exportCsvDownload()"
+                        [ladda]="exportInvoiceRequestInProcess$ | async"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html]: check this loader

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts [hunk 2]
```diff
@@ -182,7 +192,8 @@ export class AdvanceReceiptReportComponent implements AfterViewInit, OnDestroy,
         private settingsBranchAction: SettingsBranchActions,
         private modalService: BsModalService,
         private route: ActivatedRoute,
-        private invoiceBulkUpdateService: InvoiceBulkUpdateService
+        private invoiceBulkUpdateService: InvoiceBulkUpdateService,
+        private _invoiceService: InvoiceService
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts]: remove underscore

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts [hunk 3]
```diff
@@ -869,4 +880,42 @@ export class AdvanceReceiptReportComponent implements AfterViewInit, OnDestroy,
...
+        if (this.selectedReceipts?.length === 1) {
+            dataTosend.accountUniqueName = this.selectedReceipts[0].account?.uniqueName;
+        } else {
+            delete dataTosend.accountUniqueName;
+        }
+        if (this.selectedReceipts.length) {
+            dataTosend.uniqueNames = this.selectedReceipts;
+        }
+        this.exportcsvRequest.dataToSend = dataTosend;
+        this._invoiceService.exportCsvInvoiceDownload(this.exportcsvRequest).pipe(takeUntil(this.destroyed$)).subscribe((response) => {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts]: remove underscore from here also

File: apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html [hunk 4]
```diff
@@ -92,32 +92,43 @@
...
                 >
                     <button
-                        dropdownToggle
-                        type="button"
-                        class="btn btn-primary more-btn-dropdown dropdown-toggle mr-0"
-                        aria-controls="dropdown-basic"
+                        class="btn btn-default mr-r15"
+                        [tooltip]="commonLocaleData?.app_export"
+                        (click)="exportCsvDownload()"
+                        [ladda]="exportInvoiceRequestInProcess$ | async"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html]:
```suggestion
                        [ladda]="exportPaymentRequestInProcess$ | async"
```

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html]: Done

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts]: Done

File: apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html]: Removed unused variable

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**